model_cards/lit.md

```diff
@@ -0,0 +1,215 @@
+# Model Card: LiT (Locked image Tuning)
+
+Last updated: 2022-06-19
+
+Version: 1.0
+
+- This doc: https://github.com/google-research/vision_transformer/blob/main/model_cards/lit.md
+- Model Page: https://github.com/google-research/vision_transformer#lit-models
+- Other Links:
+  [LiT Blogpost](https://ai.googleblog.com/2022/04/locked-image-tuning-adding-language.html),
+  [LiT Paper],
+  [LiT Demo](https://google-research.github.io/vision_transformer/lit/)
+
+A text/image input model that can be used to embed text/image individually,
+and compute similarities between embeddings of text/image pairs. This enables
+use cases like zeroshot classification, or image/text retrieval.
+
+Note that this model card refers to the models that have been released on
+Github specifically (B16B, L16L). The [LiT Paper] also evaluates models that
+have not been released and use different datasets for training.
+
+[LiT Paper]: https://arxiv.org/abs/2111.07991
+
+## Model Summary
+
+- Architecture: Multimodal model with transformer text encoder and transformer
+  image encoder.
+- Inputs: Images presented in 224x224x3 input, text inputs are tokenized and
+  cropped to 16 tokens.
+- Outputs: Image and text embeddings (of size 768 or 1024).
+- Person of contact: Andreas Steiner (Google Brain)
+- Model authors: Xioahua Zhai, Xiao Wang, Basil Mustafa, Andreas Steiner,
+  Daniel Keysers, Alexander Kolesnikov, Lucas Beyer (Google Brain)
+
+Citation:
+
+```bibtex
+@article{zhai2022lit,
+  title={LiT: Zero-Shot Transfer with Locked-image Text Tuning},
+  author={Zhai, Xiaohua and Wang, Xiao and Mustafa, Basil and Steiner, Andreas and Keysers, Daniel and Kolesnikov, Alexander and Beyer, Lucas},
+  journal={CVPR},
+  year={2022}
+}
+```
+
+## Model Data
+
+Training data:
+
+- [Pre-trained image-tower](http://arxiv.org/abs/2106.10270)
+  - [ImageNet-21k](https://arxiv.org/abs/2104.10972)
```

Comment:
It's still unclear which checkpoints were used for the image tower in the LiT paper from the description.

Could you clarify this especially w.r.t the released checkpoints?

Reply:
We followed the recommendations from the paper (using the "recommended" checkpoints). Clarified.

Reply:
Added.